### ISSUE TITLE
Rename Repository 

### ISSUE THREAD

**Issue by egladman** [CWDG/open-source-club-website#188]
`osuosc/website` reads better than `osuosc/open-source-club-website`

**Comment by benlk** [comment]
If the repo is renamed, how does that affect people's ability to push to their established remotes that use the old URL?

**Comment by egladman** [comment]
@benlk  you just redefine the remote url in `.git/config`

**Comment by benlk** [comment]
So everyone would need to update their remotes. :(

I don't think URLs should change in a breaking way.

**Comment by egladman** [comment]
Realistically this only effects a handful of people. I'm not too concerned.

**Comment by egladman** [comment]
**Once these changes go into effect** people that have _previously_ cloned the repository will have to execute the following command.

``` bash
#ssh
git remote set-url origin git@github.com:OSUOSC/website.git

#https
git remote set-url origin https://github.com/OSUOSC/website.git
```

**Comment by egladman** [comment]
alternatively we could rename the repository to `opensource.osu.edu`

**Comment by Nefari0uss** [comment]
Honestly, it would affect maybe 3 people? Don't think it would be that bad.

**Comment by egladman** [comment]
I created a pull request #197

**Comment by moore3071** [comment]
@benlk, those who have clone it already don't **need** to change their remote urls (although they should). As explained [here](https://help.github.com/articles/renaming-a-repository/) the old remote will work unless we place a repo with the same name under the organization.

**Comment by egladman** [comment]
@moore3071 i don't care too much what the repo name is, therefore i'll go with my initial plan and name the repo `website`

**Comment by egladman** [comment]
resolved f61197202c19c3809e8cea350362f2d849687dce
